README.md

```diff
@@ -198,6 +198,20 @@ in [Endpoints.scala](https://github.com/VirtusLab/ide-probe/tree/master/core/api
 
 Note that any communication with the probe is synchronous.
 
+## Screenshots
+
+Screenshots are saved in `{IDEPROBE_SCREENSHOTS_DIR}/{IDEPROBE_TEST_SUITE}/{IDEPROBE_TEST_CASE}` directory.
```

Comment:
The PR is fine, I would just improve the docs.
The env vars that you are mentioning are implementation details that we do not need to explain. Users use ide-probe through the `driver`. Driver is responsible for installing and starting IntelliJ. The driver passes information about current test case and screenshot dirs through env vars to the IntelliJ Ide-Probe Plugin. We could pass it through system properties or some config endpoint. If someone sets `IDEPROBE_SCREENSHOTS_DIR` in CI, it should be overwritten by whatever is in `probe.paths.sceenshots`.
The only official way to set the env var is by configuring probe screenshot path in the fixture, either through the .conf file or in code. 
Additionally, thanks to HOCON feature, it is possible to populate value of any config value from env var and even make it optional. It makes sense to include this information here as you did.
Additional user facing information should be that inside screenshot dirs screenshots will be organized by test-suite/test-case if we manage to detect them.

To summarize, users should know:
-how to configure screenshot dir using official config
-how to use env vars with hocon
-what is the default for screenshot dir
-how and why are screenshots organized inside screenshots dir
and users don't need to know:
-underlying implementation passes information between driver and intellij through env vars.

Reply:
<img width="665" alt="Screenshot 2022-06-23 at 17 36 15" src="https://user-images.githubusercontent.com/33667003/175341678-25f8ec6d-1693-42d6-a74c-64a767d4c900.png">
Here should be IntellijPaths or IdeProbePaths ?

Reply:
AFAICS there are 4 methods called `withPaths` across the project (including overrides)... all accepting a param of type `org.virtuslab.ideprobe.IdeProbePaths`

Reply:
Yes and we can create IdeProbePath with some specific screenshot Path (so as i understand we need to pass some Path object there)

Reply:
Yes! a `Path` instance when configuring programatically from Scala code, or a string (converted into Path under the hood) when defined in HOCON .conf file like in git-machete-intellij-plugin